src/services/doppler-billing-user-api-client.double.ts

```diff
@@ -108,6 +109,14 @@ export const fakeDeclinedInvoices = {
   totalPending: 8.0,
 };
 
+export const fakeInvoices = {
+  invoices: [
+    { date: new Date(), invoiceNumber: '123', amount: 5, error: '', status: 'declined' },
+    { date: new Date(), invoiceNumber: '123', amount: 5, error: '', status: 'approved' },
```

Comment:
Can invoices have different `numbers` and `amounts`?

Reply:
Yes